README.md

```diff
@@ -10,6 +10,24 @@ This Emacs package provides a minor mode which configures [Magit](https://github
   <img width=500px src="https://user-images.githubusercontent.com/52205/80056404-23745500-84f2-11ea-9ecd-832376faf2f1.png" alt="image" />
 </td></tr></table>
 
+In the event that `magit-delta-mode` is your preferred mode, one can enable this in `.emacs` like so:
+
+```emacs
+(magit-delta-mode)
+```
+
+However, this has a startup cost when executing a new `emacs` session. If `emacs` has a keybinding to start `magit-diff-buffer-file` on a buffer, this function may help if added to that keybinding:
```

Comment:
Hi @kjmph what's the cost you are referring to here -- were you finding that executing `(magit-delta-mode)` was noticeably slow?

Reply:
Without `(magit-delta-mode)` in .emacs:

```
Init time was 0.38s (0.37s in .emacs).
```

With `(magit-delta-mode)` in .emacs:

```
Init time was 0.90s (0.87s in .emacs).
```

I happen to cold start emacs a lot, so this is a noticeable change for me.

Reply:
That's really interesting. I haven't profiled but I can't see anything in the [minormode function](https://github.com/dandavison/magit-delta/blob/d988abd99882c6b89f21f2746f721a4d7ece6ad4/magit-delta.el#L68-L101) that I would expect to take any time at all. Is it possible that during your init sequence magit actually displays a diff? (I know that sounds a bit unlikely)

Reply:
There shouldn't be anything in my startup that is displaying a diff, even if I open straight into `*scratch*` I see the slow down. I can try looking into it more at a later time.